<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(24, 68, 87);
background-color: rgb(24, 68, 87);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>700</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 30px;
font: bold;
color: rgb(255, 222, 222);
color: rgb(255, 151, 151);</string>
   </property>
   <property name="text">
    <string> Processes with Attributes</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>841</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(66, 220, 220);
background-color: rgb(66, 220, 220);</string>
   </property>
   <column>
    <property name="text">
     <string>PID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Process Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Memory (MB)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Memory%</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Created Time</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Threads</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>690</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 200, 31);font: 13pt &quot;Academy Engraved LET&quot;;</string>
   </property>
   <property name="text">
    <string>Processes:-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 200, 31);font: 13pt &quot;Academy Engraved LET&quot;;</string>
   </property>
   <property name="text">
    <string>12345</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 200, 31);font: 13pt &quot;Academy Engraved LET&quot;;</string>
   </property>
   <property name="text">
    <string>Running</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 200, 31);font: 13pt &quot;Academy Engraved LET&quot;;</string>
   </property>
   <property name="text">
    <string>12345</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 200, 31);font: 13pt &quot;Academy Engraved LET&quot;;</string>
   </property>
   <property name="text">
    <string>sleeping</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 200, 31);font: 13pt &quot;Academy Engraved LET&quot;;</string>
   </property>
   <property name="text">
    <string>12345</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 200, 31);font: 13pt &quot;Academy Engraved LET&quot;;</string>
   </property>
   <property name="text">
    <string>Threads</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 200, 31);font: 13pt &quot;Academy Engraved LET&quot;;</string>
   </property>
   <property name="text">
    <string>12345</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 200, 31);font: 13pt &quot;Academy Engraved LET&quot;;</string>
   </property>
   <property name="text">
    <string>Memory:-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 200, 31);font: 13pt &quot;Academy Engraved LET&quot;;</string>
   </property>
   <property name="text">
    <string>12345</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
